css/main.css

```diff
@@ -92,21 +160,17 @@ h2 {
 	padding-top: 35px;
 }
 
-.sights-heading {
-	clear: both;
-	padding-top: 20px;
-
-}
-
 input[type=radio]+label,
 .present,
 .mail {
 	display: block;
 }
 
-/* * {
-	border: 2px solid red;
-}  */
+div.present:not(:nth-child(3)) label::after,
+.tel label::after,
+.mail label::after {
+	content: '*';
```

Comment:
need to create a separate class, e.g. **.required** and add that to the obligatory inputs.